server/tserver/src/main/java/org/apache/accumulo/tserver/log/RecoveryLogsIterator.java

```diff
@@ -52,7 +52,15 @@
 
     try {
       for (Path log : recoveryLogPaths) {
-        iterators.add(new RecoveryLogReader(fs, log, start, end));
+        LOG.debug("Opening recovery log {}", log.getName());
+        RecoveryLogReader rlr = new RecoveryLogReader(fs, log, start, end);
```

Comment:
Should this reader be opened in a try-with-resources or closed in a finally block

Reply:
No, because rlr is added to a list of readers that the class keeps later in the method.

Reply:
got it, thanks